source/index.html

```diff
@@ -4,9 +4,228 @@
+              </th>
+            </tr>
+            <tr>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Розовый фильтр</span><span class="visually-hidden">Включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Розовый фильтр</span><span class="visually-hidden">Включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Розовый фильтр</span><span class="visually-hidden">Включено в тариф.</span></td>
+            </tr>
+            <tr>
+              <td class="pricing-table__field pricing-table__field--no"><span>Смайлики</span><span class="visually-hidden">Не включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Смайлики</span><span class="visually-hidden">Включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Смайлики</span><span class="visually-hidden">Включено в тариф.</span></td>
+            </tr>
+            <tr>
+              <td class="pricing-table__field pricing-table__field--no"><span>Комментарии</span><span class="visually-hidden">Не включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--no"><span>Комментарии</span><span class="visually-hidden">Не включено в тариф.</span></td>
+              <td class="pricing-table__field pricing-table__field--ok"><span>Комментарии</span><span class="visually-hidden">Включено в тариф.</span></td>
+            </tr>
+          </table>
+          <div class="prices__toggles slider-toggles">
+            <button class="slider-toggles__item" type="button" aria-label="Базовый тариф"></button>
+            <button class="slider-toggles__item slider-toggles__item--active" type="button" aria-label="Стандартный тариф" aria-current="true"></button>
+            <button class="slider-toggles__item" type="button" aria-label="Безлимитный тариф"></button>
+          </div>
+      </section>
+
+      <section class="contacts">
+        <h2 class="visually-hidden">Контактная информация.</h2>
+        <p class="contacts__slogan">Остались вопросы?<br>Свяжитесь с нами!</p>
+        <ul class="contacts__items contacts-list">
+          <li class="contacts-list__item contacts-list__item--phone">
+            <p class="contacts-list__title">
+              Звоните:
+            </p>
+            <p class="contacts-list__text">
+              <a href="tel:+78125556666">+7 (812) 555-66-66</a>
+            </p>
+          </li>
+          <li class="contacts-list__item contacts-list__item--email">
+            <p class="contacts-list__title">
+              Пишите:
+            </p>
+            <p class="contacts-list__text">
+              <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
+            </p>
+          </li>
+          <li class="contacts-list__item contacts-list__item--adress">
```

Comment:
```suggestion
          <li class="contacts-list__item contacts-list__item--address">
```

тут нужно тупо запомнить, что наш адрес с одними согласными, а их всегда с двумя aDDreSS

ну и в копилку ещё одно слово - attack - атака, у нас тоже одна т, у них две tt